### PR TITLE
Added workflows and updated .gitignore

### PR DESCRIPTION
**Pull request by eyalk007**
Added `Frogbot`, `CLA` and the `safe to test` workflows.
Also updated `.gitignore` to not include pem files as they are in constant use in this project.

### CODE REVIEW

File: .github/workflows/frogbot-scan-pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          # SMTP server URL including should the relevant port: (Example: smtp.server.com:8080)
+          JF_SMTP_SERVER: ${{ secrets.JF_SMTP_SERVER }}
+
+          # [Mandatory if JF_SMTP_SERVER is set]
+          # The username required for authenticating with the SMTP server.
+          JF_SMTP_USER: ${{ secrets.JF_SMTP_USER }}
+
+          # [Mandatory if JF_SMTP_SERVER is set]
+          # The password associated with the username required for authentication with the SMTP server.
+          JF_SMTP_PASSWORD: ${{ secrets.JF_SMTP_PASSWORD }}
```
Comment (eyalk007) [inline .github/workflows/frogbot-scan-pull-request.yml:41]: i dont think we need this part about `SMPT`

Comment (eyalk007) [inline .github/workflows/frogbot-scan-pull-request.yml:41]: i dont think we need this part about `SMPT`

Comment (hadarshjfrog) [reply .github/workflows/frogbot-scan-pull-request.yml:41]: we don't - you can remove

File: .github/workflows/frogbot-scan-repository.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  scan-repository:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # The repository scanning will be triggered periodically on the following branches.
+        branch: [ "dev" ]
+    steps:
+      - uses: jfrog/frogbot@v2
+        env:
+          JFROG_CLI_LOG_LEVEL: "DEBUG"
```
Comment (hadarshjfrog) [inline .github/workflows/frogbot-scan-repository.yml:21]: are you sure you want DEBUG every time?
Not objecting - your call...

File: .github/workflows/frogbot-scan-pull-request.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+jobs:
+  scan-pull-request:
+    runs-on: ubuntu-latest
+    # A pull request needs to be approved before Frogbot scans it. Any GitHub user who is associated with the
+    # "frogbot" GitHub environment can approve the pull request to be scanned.
+    environment: frogbot
+    steps:
+      - uses: jfrog/frogbot@v2
+        env:
+          JFROG_CLI_LOG_LEVEL: "DEBUG"
```
Comment (hadarshjfrog) [inline .github/workflows/frogbot-scan-pull-request.yml:17]: are you sure you want DEBUG every time?
Not objecting - your call...

File: .github/workflows/frogbot-scan-repository.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  contents: write
+  pull-requests: write
+  security-events: write
+jobs:
+  scan-repository:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # The repository scanning will be triggered periodically on the following branches.
+        branch: [ "dev" ]
```
Comment (hadarshjfrog) [inline .github/workflows/frogbot-scan-repository.yml:17]: "dev" only? no "main"?
again - I'm ok with it - just need to align on our standards on cli-security

File: .github/workflows/frogbot-scan-repository.yml (hunk 2 above)
Comment (eyalk007) [reply .github/workflows/frogbot-scan-repository.yml:21]: this is what we do in other projects like the cli

**Review by hadarshjfrog — APPROVED** [review]

**Pull request merged**